lib/connect.c

```diff
@@ -472,6 +472,8 @@ static CURLcode bindlocal(struct Curl_easy *data,
     if(--portnum > 0) {
       infof(data, "Bind to local port %hu failed, trying next", port);
       port++; /* try next port */
+      if(port == 0)
+        break;
```

Comment:
Maybe it would be nicer to have the infof() done after the check? "trying next" might turn out wrong otherwise.

Reply:
Good point there, fixed that.